src/commands/Admin/exec.ts

```diff
@@ -1,36 +1,33 @@
-import { Attachment, Message } from '@klasa/core';
-import { AlestraCommand, AlestraCommandOptions } from '@lib/structures/AlestraCommand';
-import { PermissionLevels } from '@lib/types/Enums';
+import { ApplyOptions } from '@sapphire/decorators';
+import { Args, Command, CommandOptions } from '@sapphire/framework';
 import { codeBlock } from '@sapphire/utilities';
-import { ApplyOptions } from '@skyra/decorators';
 import { exec } from '@utils/exec';
 import { fetch, FetchMethods, FetchResultTypes } from '@utils/util';
+import { Message, MessageAttachment } from 'discord.js';
 
-@ApplyOptions<AlestraCommandOptions>({
+@ApplyOptions<CommandOptions>({
 	aliases: ['execute'],
-	description: (language) => language.get('COMMAND_EXEC_DESCRIPTION'),
-	extendedHelp: (language) => language.get('COMMAND_EXEC_EXTENDED'),
-	guarded: true,
-	permissionLevel: PermissionLevels.BotOwner,
-	usage: '<expression:string>',
-	flagSupport: true
+	description: 'COMMAND_EXEC_DESCRIPTION',
```

Comment:
Replace this with an actual description please. I know it doesn't mean much but this is just ugly.